### PR TITLE
Add April post

### PR DESCRIPTION
**Pull request by d-m-u**
:)

### CODE REVIEW

**Comment by JPrause** [comment]
LGTM!  But wait,....you know about the movie Airplane???? Wow! Just wow. :wink:

**Comment by JPrause** [comment]
@jrafanie @lgalis can you review. cc @cybette

File: site/_posts/2018-05-02-last-week-in-manageiq-two-mcminutes.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  
+You wake up. The dog is nowhere to be seen. Incidentally, your tool to process logs isn't either, and you wonder ruefully if you should have saved it from its fate. The timer is counting down. You grab it and run, heading south again until you reach your catalog item with a tag control element from yesterday. And lo and behold, it appears [to be fully functional](https://github.com/ManageIQ/ui-components/pull/278) today! Marvelling, you barely notice that you're again out of time.  
+  
+You wake up. The dog is back. Your process logs tool is beaten up but still mostly intact. You grab it and the timer. The dog stirs but does not wake. You hide the tool in your chest, delighting at the fact that it also contains a new [Openstack Cinder EventCatcher worker](https://github.com/ManageIQ/manageiq/pull/17351) and head out again with the timer. This time there appears to be a small river next to your house. You head towards the sound of the water. On the beach are several [RVH markers](https://github.com/ManageIQ/integration_tests/pull/7161). Delighted, you grab them. Ding!
```
Comment (syncrou) [inline site/_posts/2018-05-02-last-week-in-manageiq-two-mcminutes.md]: @d-m-u - Do you mean `RHV` instead of `RVH` here?

**Comment by JPrause** [comment]
@jrafanie TMI Joe, TMI. :smile:

**Comment by cybette** [comment]
LGTM! Constantly amazed at @d-m-u 's imagination and creativity :smile: :+1:

**Pull request merged**